source/plugin/data/serialization.rst

```diff
@@ -123,3 +123,62 @@ from persistent files using the :doc:`Configurate Library <../configuration/inde
         final DataView container = translator.translate(node);
         return deserializeHealth(container);
     }
+
+
+DataFormat
+==========
+A alternative to DataTranslators is using :javadoc:`DataFormat.readFrom` or :javadoc:`DataFormat.writeTo` which allows you to convert a data container this allows you to store a data container as a hocon, json or even NBT you can of course retrive data using data format.
```

Comment:
You have to add your class to the javadoc-import block at the top of the page. Also I'm not sure whether you can reference methods like this.